### PR TITLE
added some news sources to list for tvnews

### PR DESCRIPTION
**Pull request by MaxReinisch**
This required that I slightly change the way the news hosts are identified.

### CODE REVIEW

**Comment by MaxReinisch** [comment]
Your first point: This checks any url that contains ".forbes." for example, as the way the hostname is extracted is with a str.split(".").  The previous way included "www." , but that seems unnecessary.  Other than that, I don't understand how this functionality is different than how it was previously, and how you'd prefer it.  Do you want to include the ".com" or ".org" ?

Regarding point 2: Good point, I'll make that change.

**Comment by vbanos** [comment]
OK, I missed that about #1.
Then, please rename the variable name. Since you extract the `hostname`, the variable name should be `final_hostname`, it shouldn't be `final_url`.

Also, please list the sites in alphabetical order, this will make them easier to read.
This becomes more helpful as the list grows longer.

**Comment by MaxReinisch** [comment]
Done

File: scripts/popup.js [hunk 1]
```diff
@@ -332,14 +332,27 @@ function borrow_books(){
 function show_news(){
     chrome.tabs.query({active: true,currentWindow:true},function(tabs){
         url=tabs[0].url;
-        var to_check_url=url.replace(/^https?:\/\//,'');
-        to_check_url = to_check_url.replace(/\.html/, "");
-        var final_url=to_check_url.slice(0,to_check_url.lastIndexOf(".")+1);
+        var to_check_url=url.replace(/^https?:\/\/(www\.)?/,'');
+        // to_check_url = to_check_url.replace(/\.html/, "");
+        var news_host=to_check_url.split(".")[0]
```
Comment (vbanos) [inline scripts/popup.js]: missing `;`

File: scripts/popup.js [hunk 2]
```diff
@@ -332,14 +332,27 @@ function borrow_books(){
...
+          "snopes",
+          "theverge",
+          "usatoday",
+          "vox", 
+          "washingtonpost",
+        ]);
         chrome.storage.sync.get(['news'],function(event){
             if(event.news==true){
-                if(list_of_sites.indexOf(final_url)>=0){
+              console.log(set_of_sites.has(news_host));
```
Comment (vbanos) [inline scripts/popup.js]: Not necessary

File: scripts/popup.js [hunk 3]
```diff
@@ -332,14 +332,27 @@ function borrow_books(){
 function show_news(){
     chrome.tabs.query({active: true,currentWindow:true},function(tabs){
         url=tabs[0].url;
-        var to_check_url=url.replace(/^https?:\/\//,'');
-        to_check_url = to_check_url.replace(/\.html/, "");
-        var final_url=to_check_url.slice(0,to_check_url.lastIndexOf(".")+1);
+        var to_check_url=url.replace(/^https?:\/\/(www\.)?/,'');
+        // to_check_url = to_check_url.replace(/\.html/, "");
```
Comment (vbanos) [inline scripts/popup.js]: Do we need to keep this?

**Pull request merged**